.github/workflows/charterafrica-deploy-prod.yml

```diff
@@ -86,7 +86,7 @@ jobs:
           cache-to: type=local,dest=/tmp/.buildx-cache-new
           context: .
           file: ./Dockerfile.charterafrica
-          platforms: linux/arm64
+          platforms: linux/amd64
```

Comment:
```suggestion
```

Isn't `linux/amd64` the default (at least charterafrica-deploy-dev.yml thinks o).